esp/src/src/store/Store.ts

```diff
@@ -21,7 +21,7 @@ export interface QueryOptions<T> {
     start?: number;
     count?: number;
     sort?: QuerySort<T>;
-    alphanumColumns: { [id: string]: boolean };
+    alphanumColumns?: { [id: string]: boolean };
```

Comment:
I said it was one line change, but I did also make this optional, which is perhaps incorrect. It would've required defining options as `{ alphanumColumns: {} }` instead.